docs/content/preview/yugabyte-platform/security/enable-encryption-in-transit.md

```diff
@@ -247,13 +247,31 @@ If the key is protected by a passphrase in the PKCS12 archive, you are prompted
 
 #### Verify certificate chain
 
-To verify the certificate chain, execute the following command:
+Perform the following steps to verify your certificates:
 
-```sh
-openssl verify -CAfile bundle.pem cert.pem
-```
+1. Execute the following verify command which checks the database node certificate "node.crt" against the root CA certificate "ca.crt":
+
+    ```sh
+    openssl verify -CAfile bundle.pem cert.pem
```

Comment:
minor: Given we refer to this as ca.crt and node.crt above, we could use the same naming here.